packages/web/components/Cards.tsx

```diff
@@ -1,16 +0,0 @@
-import { ICardOld } from "@/lib/api";
```

Comment:
I think this should be kept, but I'm not sure. Generally there's probably not usually a reason for this PR to delete things.